sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuFileSourceScanExec.scala

```diff
@@ -69,8 +70,9 @@ case class GpuFileSourceScanExec(
     extends GpuDataSourceScanExec with GpuExec {
   import GpuMetric._
 
-  private val isParquetFileFormat: Boolean = relation.fileFormat.isInstanceOf[ParquetFileFormat]
-  private val isPerFileReadEnabled = rapidsConf.isParquetPerFileReadEnabled || !isParquetFileFormat
+  // CSV should be always using PERFILE read type
+  private val isPerFileReadEnabled = rapidsConf.isParquetPerFileReadEnabled ||
+    rapidsConf.isOrcPerFileReadEnabled || relation.fileFormat.isInstanceOf[CSVFileFormat]
```

Comment:
this might be safer to just put (! orc or parquet) rather then csv, so that if we add json or other types it will default to the per file.

Reply:
Done